pkg/ui/workspaces/cluster-ui/src/sqlActivity/util.tsx

```diff
@@ -90,13 +92,18 @@ export function filterStatementsData(
         return databases.length === 0 || databases.includes(statement.database);
       }
     })
-    .filter(
-      statement =>
-        !appNames?.length ||
-        appNames.includes(
+    .filter(statement => {
+      const isInternal = statement.applicationName?.startsWith(
+        INTERNAL_APP_NAME_PREFIX,
+      );
+      return (
+        (!appNames?.length && !isInternal) ||
```

Comment:
just to make sure i understand this line: is it saying "if no app names were selected in the dropdown, then this will only show non-internal queries" ?

Reply:
That's correct.